branding/celo/src/icons/LogoHeart.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from 'react'
```

Comment:
this is actually just the Logo file, i figure there's no need to create a specific heart version of the non-branded logo